<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>亚电新能源写卡程序-NFC_Pn532</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0,0">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>串口号:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenComm">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_comm_state">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>卡类型</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioStartStop">
          <property name="text">
           <string>起停卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioCorpNetwork">
          <property name="text">
           <string>公司网络卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioCorpOffline">
          <property name="text">
           <string>公司离线卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioStationManage">
          <property name="text">
           <string>场站管理卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioStationNetwork">
          <property name="text">
           <string>场站网络卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioStationOffline">
          <property name="text">
           <string>场站离线卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSingleDevice">
          <property name="text">
           <string>单桩启停卡</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>卡参数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>充电站ID:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStationID">
            <property name="maxLength">
             <number>10</number>
            </property>
            <property name="placeholderText">
             <string>长度不能超过10位</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>有效期截止日期：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditStopDate">
            <property name="currentSection">
             <enum>QDateTimeEdit::YearSection</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>充值金额:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxMoney">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>元</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>充电桩SN:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditDeviceSN">
            <property name="maxLength">
             <number>16</number>
            </property>
            <property name="placeholderText">
             <string>长度不能超过16位</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>60</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnWrite">
       <property name="text">
        <string>写卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
